CONTRIBUTING.md

```diff
@@ -1,3 +1,7 @@
-List of contribute actions people to develop this project
-Co-authored-by: Thuong Truong <https://github.com/thuongtruong1009/>
-Co-authored-by: Codev Ta <https://github.com/font-end-pro/>
+#### List of contribute actions people to develop this project Co-authored-by:
+| Number | Name | User | Role|
+|---|---|---|---|
+| 1 | Tran Nguyen Thuong Truong | [thuongtruong1009](https://github.com/thuongtruong1009) | fix, add, PR, merge |
+| 2 | Codev Ta | [font-end-pro](https://github.com/font-end-pro/) | issue, disscuss, PR |
+
```

Comment:
If you want contributing, you should add info add this table then easy to see badge on this repository